lib/charms/karma_k8s/v0/karma.py

```diff
@@ -0,0 +1,269 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+""" # Karma library
+
+This library is designed to be used by a charm consuming or providing the karma-dashboard relation.
+"""
+
+import logging
+
+import ops.charm
+from ops.framework import EventBase, EventSource, ObjectEvents
+from ops.charm import RelationJoinedEvent, RelationDepartedEvent
+from ops.relation import ConsumerBase, ProviderBase
+from ops.framework import StoredState
+
+from typing import List, Dict, Optional, Iterable
+
+# The unique Charmhub library identifier, never change it
+LIBID = "abcdef1234"
+
+# Increment this major API version when introducing breaking changes
+LIBAPI = 0
+
+# Increment this PATCH version before using `charmcraft publish-lib` or reset
+# to 0 if you are raising the major API version
+LIBPATCH = 2
```

Comment:
Should this be `2`?

Reply:
Was handed over to me like that, I think. Should probably be kept monotonic and rising even if not published yet.
Unless there is a reason to revert back to 1?

Reply:
I don't think we should revert it. Although blast radius of this is probably restricted to Xav :-)